src/applications/proxy-rewrite/README.md

```diff
@@ -25,3 +25,34 @@ http://localhost:3001/?target=https://www.va.gov/health/
 
 ## Charles Proxy
 You can also use an application called Charles Proxy to map the `proxy-rewrite` bundles of TeamSite pages to your local machine. This way you can navigate directly to `https://www.va.gov/health/` and when the request for the production bundle of `proxy-rewrite` is sent, Charles will have overridden that file to instead be served locally. Instructions to set this up are located here, https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Engineering/Teamsite.md.
+
+## Teamsite Visual Regression Testing
+
+Both scripts compare production Teamsite header and footer against images in `vets-website` repo
+`npm run vrt` - test fails and generates diff images when images differ
+`npm run vrt:baseline` - test always succeeds and replaces existing baseline images when images differ
+
+- If no baseline images exist for a test, both scripts will
+  - succeed on each test that has no baseline image and
+  - generate a new baseline image for it.
+- Diff images are removed automatically when their associated test succeeds.
+
+### Details
+- compares snapsshots of `<header>` and `.footerNav` on current subdomain Teamsite pages
```

Comment:
`<footer>` includes an updated date which will cause the tests to fail everyday. Opted to exclude that part from the baseline image rather than try to tweak the `pixelmatch` tolerances.